app/controllers/projects_controller.rb

```diff
@@ -66,10 +66,26 @@ def user_list_params(counter, key_prefix)
       users.compact.uniq
     end
 
-    def form_metadata
+    def project_timestamps(project:)
+      timestamps = {}
+      # TODO: Use the correct dates instead of the hard coded ones
+      if project.nil?
+        timestamps[:created_by] = current_user.uid
+        timestamps[:created_on] = DateTime.now.strftime("%d-%b-%Y %H:%M:%S")
```

Comment:
Awesome! I am glad you were able to figure out the right incantation for this format!

Reply:
🎉🎉